bigchaindb/commands/bigchain.py

```diff
@@ -146,7 +146,13 @@ def run_init(args):
 def run_drop(args):
     """Drop the database"""
     bigchaindb.config_utils.autoconfigure(filename=args.config, force=True)
-    db.drop(assume_yes=args.yes)
+    try:
+        db_name = 'Unknown database'
+        if db != None: db_name = db.get_database_name()
+        db.drop(assume_yes=args.yes)
+    except DatabaseDoesNotExist:
+        print('Cannot drop \'{name}\'. The database does not exist.'.format(name=db_name), file=sys.stderr)
+        print('Consider initializing the database before attempting drop.', file=sys.stderr)
```

Comment:
I'm referring to L155 `print('Consider initializing the database before attempting drop.', file=sys.stderr)`, you can remove it since it's not adding much information to the previous message.